kiwi/builder/live.py

```diff
@@ -240,10 +242,11 @@ def create(self):
             if self.firmware.efi_mode() == 'uefi':
                 # write bootloader config to EFI directory in order to allow
                 # grub loaded by shim to find the config file
-                shutil.copy(
-                    self.media_dir + '/boot/grub2/grub.cfg',
-                    self.media_dir + '/EFI/BOOT'
+                grubconf_glob = os.sep.join(
+                    [self.media_dir, 'boot/grub*/grub.cfg']
```

Comment:
What happens if both paths exist, which can happen for a variety of reasons? This seems like it'd work out pretty badly. You should really test for the versioned one first, and then the unversioned one if the former doesn't exist.